### PR TITLE
Added new OS FAQ topics

### PR DESCRIPTION
**Pull request by george-hoffer**

### CODE REVIEW

File: modules/ROOT/pages/osv2-faq.adoc [hunk 1]
```diff
@@ -31,6 +31,20 @@ Yes, Object Store v2 is in the same region as the worker where the application i
 
 IMPORTANT: If after the first deploy, you move the app to a different region, Object Store v2 remains in the original region to avoid data loss. Your use of Object Store v2 never moves from one region to another. When you need to deploy both an app and object store to a different region, delete the application and re-upload the app into the new region. This action loses all the data that may exist in the storage.
 
+== How are Object Store entries divided among workers?
+
+All entries that you put in `_defaultUserObjectStore` are available to all workers for the same app. For example, if you had 50 entries and 5 workers, all 50 entries would be available to all the workers in the app and the Object Store v2 service handles storing all the data without any impact on the workers.
```
Comment (steveCluster) [inline modules/ROOT/pages/osv2-faq.adoc]: I would add something here reminding people that OSv2 is a remote service that is separate from the workers.

File: modules/ROOT/pages/osv2-faq.adoc [hunk 2]
```diff
@@ -31,6 +31,20 @@ Yes, Object Store v2 is in the same region as the worker where the application i
...
+
+== Can Object Store v2 be shared between workers?
+
+An Object Store and its contents (keys, values) are accessible to all CloudHub workers of the one Mule application that defines and therefore owns that Object Store. This means that as long as CloudHub workers are of the same application, they can share an Object Store.
```
Comment (steveCluster) [inline modules/ROOT/pages/osv2-faq.adoc]: We should mention here that only when using the `_defaultUserObjectStore` doest the data persist in OSv2 and is therefore available to all workers.  If not, then the data is put into an in-memory object store on the worker and will be gone when the worker is restarted or a new app version is deployed.

Comment (george-hoffer) [reply modules/ROOT/pages/osv2-faq.adoc]: Changed topic wording to:

**How are Object Store entries divided among workers?**

Object Store v2 is a remote service that is separate from CloudHub workers. All entries that you put in `_defaultUserObjectStore` are available to all workers for the same app. When using `_defaultUserObjectStore`, data persists in Object Store v2 and is therefore available to all workers. If not using `_defaultUserObjectStore`, then the data is put into an in-memory object store on the worker and is lost if the worker restarts or a new app version is deployed.

For an example of entry availability, if you had 50 entries and 5 workers, all 50 entries would be available to all the workers in the app and the Object Store v2 service handles storing all the data without any impact on the workers.

File: modules/ROOT/pages/osv2-faq.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/osv2-faq.adoc]: See comment below for changed wording

File: modules/ROOT/pages/osv2-faq.adoc (hunk 2 above)
Comment (george-hoffer) [reply modules/ROOT/pages/osv2-faq.adoc]: Revised wording:
**How are Object Store entries divided among workers?**

Object Store v2 is a remote service that is separate from CloudHub workers. All entries that you put in `_defaultUserObjectStore` are available to all workers for the same app. If not using `_defaultUserObjectStore`, then the data is put into an in-memory object store on the worker and that data is lost if the worker restarts or a new app version is deployed.

For an example of entry availability, if you had 50 entries and 5 workers, all 50 entries would be available to all the workers in the app and the Object Store v2 service handles storing all the data without any impact on the workers.

**How can one Mule app access the object store of another app?**

When other Mule apps need access to an object store, use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/object-store-v2/[Object Store v2 REST API] to read from one app and store to another.

**Review by steveCluster — APPROVED** [review]

**Pull request merged**